pixi.toml

```
@@ -53,8 +53,8 @@ initialize-julia = { depends_on = [
 # Docs
 build-julia-docs = { cmd = "julia --project docs/make.jl", depends_on = [
     "initialize-julia",
-] }
-quartodoc-build = { cmd = "quartodoc build && rm objects.json", cwd = "docs" }
+], inputs = ["core"] , outputs = ["docs"]}
```

```suggestion
], inputs = ["core"] , outputs = ["docs/build"]}
```

You can see this in the URL here: https://deltares.github.io/Ribasim/build/